pinc/page_controls.inc

```diff
@@ -0,0 +1,152 @@
+<?php
+include_once($relPath.'stages.inc'); // $Round_for_round_id_
+include_once($relPath.'metarefresh.inc');
+
+function get_page_data()
```

Comment:
Since this is returning a JS string and not an object, `get_page_data_js()` would be more accurate.

Reply:
changed this.